test/dadbot-test.coffee

```diff
@@ -31,12 +31,26 @@ describe 'DadBot', ->
   it 'responds to "I AM HuNgRy"', ->
     @room.user.say('bob', 'I AM HuNgRy').then =>
       expect(@room.messages).to.eql [
-        ['bob', 'I AM HuNgRy'],
```

Comment:
Is this a coffeescript thing (not needing a comma between arrays if they're separated by a line)?

Reply:
yeah, I just tweaked that so the styling would fit better with the other tests